### DISCUSSION TITLE
Mesh optimization/smoothing

### DISCUSSION THREAD

**Discussion by tile-timley** [DHI/mikeio#367] [Ideas]
I think it would be very interesting to test if a mesh can be smoothed by using MIKIEIO in conjunction with specifically the python package ["OPTIMESH"](https://github.com/meshpro/optimesh).

I am by no means a skilled scripter or have extensive experience in working with mesh optimization, but I think it looks like there could be potential to use the two packages together. When I have the time I will give it a try myself, but I wanted to pass on the thought/idea to more skilled scripters.

**Comment by jsmariegaard** [comment]
Thanks a lot, @tile-timley! Sounds like a good idea.
